tests/acceptance/FrontendTest.php

```diff
@@ -24,6 +24,33 @@ protected function setUp()
         }
     }
 
+    /**
+     * Reset columns if the exits.
+     *
+     * @param int $shopId
+     */
+    public function activateModules($shopId = 1)
+    {
+        if (method_exists($this, 'executeSql')) {
+            $aTableColumns = array(
```

Comment:
Are this fields not used anymore? What happens if we drop this fields on activation?

Reply:
OXID restores the database on every test but keeps the added columns, that results in a mysql error if the module was activated once for a previous test. So we remove these columns if the exist already that we can activate the module again.